README.md

```diff
@@ -1,2 +1,53 @@
-# rhcos-slb
-Config scripts for RHCOS SLB 
+# Setting up network bonding on specific interfaces during the ignition process
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+This guide describes the customisations needed for installation when using a PXE server.
+
+## Prerequisites
+- [Fcct](https://github.com/coreos/butane) software to be able to convert fcc to ign files.
+- PXE server with ability to set kernel arguments.
+- Web accessible location (HTTP only) to upload the custome ign files.
+
+## Usage  
+
+### 1. Adding parameters to PXE server
+Key   |  Type | Value | Exmaple |Discription
+---   | --- | --- | --- | --- |
+ignition.config.url  | String | http://{location reachable to PXE and remote VM in order to get the ignition file}/```file.ign``` custom-config | ignition.config.url=```http://my.web.server.redhat.com/file.ign``` custom-config | This parameter is used to allow the server to load an extra ignition file which captures the MAC addresses and saves them into a file for later use in  machine config.
+macAddressList | String | List of MAC address should be set per host In the following order:  1. Primary NIC  2. Secondary NIC  3. All the rest (optional)| 00:01:02:03:04:05,06:07:08:09:10:11 ... | This will be the list of MAC addresses which exist on the host and will be grabbed by the ignition file.
+
+
+### 2. Prepare the files
+- Upload `file.ign` to a shared location which the OpenShift nodes can access.
+
+- Base64 encode the `setup-ovs.sh` file and paste the content into each MCO file into "base64_script_content" section. 
+
+    **TIP:** To update the content you can use:
+
+      ```
+      export base64_script_content=$(cat setup-ovs.sh|base64) & envsubst < mco_ovs_workers.yml|tee mco_ovs_workers.yml & envsubst < mco_ovs_supervisor.yml|tee mco_ovs_supervisor.yml
```

Comment:
It should be `&&`, `&` operator is to fork shell process and is not what you want to do.

We also need to run `base64 -w 0` to make sure we don't wrap lines in the rendered yaml.

Reply:
Done

Reply:
@phoracek 
Question, Should we call the files in the repo: mco_ovs_worker_template.yml & mco_ovs_supervisor_template.yml
and the output of the command will be mco_ovs_worker.yml & & mco_ovs_supervisor.yml ?

Reply:
That sounds right